client/components/popover/index.jsx

```diff
@@ -381,6 +367,10 @@ class Popover extends Component {
 	}
 
 	show() {
+		this.bindEscKeyListener();
+		this.bindDebouncedReposition();
+		bindWindowListeners();
```

Comment:
The `show` can be delayed by `showDelay`. Installing the listeners here is probably harmless, but I think it's more correct to install them at the time when `state.show` changes.

That means checking the state in `componentDidMount`, `componentDidUpdate` (it gets `prevState` as second argument) and uninstalling them in `componentWillUnmount`.

Reply:
Updated in edf40b1.

> uninstalling them in componentWillUnmount

I'm also uninstalling them in `hide` as we talked about before.